### DISCUSSION TITLE
Load 120k row data 8s in table resource

### DISCUSSION THREAD

**Discussion by AFSDARIF** [filamentphp/filament#13315] [General]
Is there any other way to speed up the process of loading 120k data ?

the condition is now I have used ->deferLoading()

**Comment by xalabama** [comment]
What about caching your query, and update the cache every x hours? This should improve the speed.

**Reply by AFSDARIF** [reply]
I don't have any experience with cache queries.  Maybe I'll try it now too.

**Comment by mgkimsal** [comment]
@AFSDARIF could you be more specific?  are you loading 120k rows in to a filament table and trying to display all 120k on the screen at the same time?

**Reply by AFSDARIF** [reply]
it doesn't display 120k data all at once.  I use pagination 10 and 50. it takes 8s loading time.

**Reply by llamorinlabs** [reply]
> it doesn't display 120k data all at once. I use pagination 10 and 50. it takes 8s loading time.

Probably your index
